documentation/src/main/wiki/ActionAdapter.md

```diff
@@ -0,0 +1,16 @@
+Action Adapter is Component of a system, that mediate communication between Knot.x (ActionKnot) and external 
+Services that are responsible for handling form submitting.
+
+Action Adapter is a project custom part of a system, however Knot.x comes with default `HttpActionAdapter`.
+Action Adapter entry are:
+
+- `ClientRequest` - object with all data of an original request,
+- `params` - all additional params defined in configuration.
+
+Action Adapter result contract is Json Object with fields:
```

Comment:
Result generated by Action Adapter must be a JsonObject with the fields as below: